doc/releases/release-notes-3.4.rst

```diff
@@ -1229,6 +1229,9 @@ Networking
   * Added support for handling multiple incoming TCP sessions.
   * Made zperf thread priority and stack size configurable.
   * Several minor cleanups in the module.
+=======
+  * TWT intervals are changed from milli-seconds to micro-seconds, interval variables are also renamed.
+  * Added Passive scan support. Scan API updated with `wifi_scan_params` to pass scan type.
```

Comment:
What is TWT?
Why add on 3.4 release?

Reply:
TWT - Target Wake Time

@ajayparida You should update https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/release-notes-3.5.rst instead, we're already past 3.4.

Reply:
Thanks, Just updated.